tools/test-fwe.sh

```diff
@@ -5,16 +5,23 @@
 set -eo pipefail
 
 WITH_ROS2_SUPPORT="false"
+IGNORE_TESTS=""
 
 parse_args() {
     while [ "$#" -gt 0 ]; do
         case $1 in
         --with-ros2-support)
             WITH_ROS2_SUPPORT="true"
+            shift
+            ;;
+        --extra-ctest-args)
+            IGNORE_TESTS=$2
+            shift
             ;;
         --help)
             echo "Usage: $0 [OPTION]"
             echo "  --with-ros2-support  Test with ROS2 support"
+            echo "  --extra-ctest-args Gives the tests to ignore"
```

Comment:
Description is not accurate. This are args to be passed to ctest.

Reply:
Done